website/src/main/tut/docs/users/installation.md

```diff
@@ -129,6 +129,69 @@ Once the scalafix cli is installed, consult the --help page for further usage in
 ```tut:evaluated:plain
 println(scalafix.cli.Cli.helpMessage)
 ```
+## scalafix with scala-maven-plugin based project
+
+There is some gotcha you will have to take care of when using scalafix-cli with a maven based project.
+
+### semanticdb installation
+
+Nothing is automatic here, so you will have to download `semanticdb` which correspond to you *exact* scala version, down to minor. 
+For example, for scala `2.12.3`:
+
+```
+coursier fetch --intransitive org.scalameta:semanticdb-scalac_2.12.3:2.0.1
```

Comment:
```
coursier fetch --intransitive org.scalameta:semanticdb-scalac_{{ site.scala212 }}:{{ site.scalametaVersion }}
```

Reply:
It's important to not hardcode the scala version since it changes regularly, semanticdb-scalac only supports the latest patch version so now that 2.12.4 is out we will require that version with the next release.

Reply:
just to expand on how this works, variables like `{{ site.scala212 }}` are automatically dynamically expanded while building the website (based on current sbt settings)

Reply:
I saw them, but wasn't sure of their meaning. Thanks for the explanation. Is there a list of available variable somewhere?

Reply:
https://github.com/scalacenter/scalafix/blob/eab41e5034f998b11bcc4470de2c7f5f8496cc30/project/ScalafixBuild.scala#L146-L158